encryption_works.md

```diff
@@ -254,21 +254,23 @@ In addition to protecting PGP secret keys, you also need to choose good passphra
 
 ### Software
 
-To install GPG, Windows users can download [Gpg4win](http://www.gpg4win.org/), and Mac OS X users can download [GPGTools](https://gpgtools.org/). If you run GNU/Linux you should already have GPG installed. GPG is a command line program, but there's software that interfaces with email clients that makes it much easier to use.
+To install GPG, Windows users can install [Gpg4win](http://www.gpg4win.org) and Mac OS X users can download [GPGTools](https://gpgtools.org). You'll also have to download a desktop e-mail client to use PGP correctly. This is because PGP does not yet function satisfactorily in a web browser. Freedom of the Press recommends using [Thunderbird](https://www.mozilla.org/en-US/thunderbird) as your desktop e-mail client with the [Enigmail](https://www.enigmail.net/home/index.php) add-on. 
 
-You'll have to download an email client to use PGP correctly. An email client is a program on your computer that you open to check your email, as opposed to using your web browser. The most popular PGP setup is the email client [Thunderbird](https://www.mozilla.org/en-US/thunderbird) with the [Enigmail](http://enigmail.net/home/index.php) add-on. Thunderbird and Enigmail are free software and run on Window, Mac, and GNU/Linux.
+### Sending And Receiving Encrypted E-mail
 
-Right now PGP is very difficult to use securely from a web browser. While some browser extensions exist that help with this, I would recommend sticking to a desktop email client until the field of browser crypto matures. It's possible to use PGP encryption with Gmail, but the easiest way is to set up an email client like Thunderbird and run your Gmail account through it.
+Once you've installed Enigmail, you'll notice some new settings when you go to compose an e-mail in Thunderbird.
 
-### Encrypting, Decrypting, and Signatures
+![Sending an encrypted e-mail with Thunderbird and the Enigmail add-on.](http://i.imgur.com/dnI6byg.png)
 
-You can send encrypted emails and digitally sign them using the graphical user interface provided by Thunderbird and Enigmail. Here's an example of an encrypted email that I'm sending to myself. When I hit send, my software took the body of the message and encrypted it using my public key, making the content unintelligible to eavesdroppers, and indeed to my email provider too.
```

Comment:
Is there a reason to delete this paragraph? I think it helps with the explanation and is colorful language that will grab a reader.